scanner/indexer/indexer.go

```diff
@@ -571,6 +590,35 @@ func createManifestDigest(hashID string) (claircore.Digest, error) {
 	return d, nil
 }
 
+func (i *localIndexer) StoreIndexReport(ctx context.Context, hashID string, scannerVersion string, report *claircore.IndexReport) error {
+	ctx = zlog.ContextWithValues(ctx, "component", "scanner/backend/indexer.StoreIndexReport")
+
+	sv, err := pkgscanner.DecodeVersion(scannerVersion)
```

Comment:
Is the decode here necessary? (same for the decode in the compare function) - IIUC we are sending and storing the indexer version only (not encoded version).